sigstore-gradle/sigstore-gradle-sign-base-plugin/src/main/kotlin/dev/sigstore/sign/SigstoreSignature.kt

```diff
@@ -25,7 +25,7 @@ import javax.inject.Inject
 
 abstract class SigstoreSignature @Inject constructor(private val name: String) : Named {
     companion object {
-        const val EXTENSION = "sigstore"
+        const val EXTENSION = ".sigstore.json"
```

Comment:
Apache commons-io does not include dot in the extension: https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FilenameUtils.html#getExtension(java.lang.String)

Maven does not include the dot in the extension too: https://maven.apache.org/repositories/artifacts.html

`EXTENSION` is public, so we might exclude the dot from it. WDYT?

Reply:
Sounds good. I'll adjust.